src/Forms/Main.cs

```diff
@@ -336,20 +336,6 @@ private void SetCurrentFormat(SubtitleFormat format)
             }
         }
 
-        private SubtitleFormat FormatNameToFormat(string subtitleFormatName)

```

Comment:
Avoid polluting main.cs, also makes more sense to have this method inside SubtitleFormat.cs